openfisca_us/tests/policy/baseline/gov/states/ca/tax/income/deductions/standard/ca_standard_deduction

```diff
@@ -0,0 +1,17 @@
+- name: Single filer 
```

Comment:
I think this file needs a `.yaml` extension

Reply:
Done